installers/linux/alpine/tar/build.gradle

```
@@ -108,6 +110,24 @@ task createTestImage(type: Exec) {
     commandLine 'make','test-image'
 }
 
+task bundleThirdPartyBinaries {
+    dependsOn executeBuild
+    doLast {
+        // Copy pre-populated third party binaries if present
+        if (file(thirdPartyBinaries).exists()) {
```

Can we have this use a flag for the location and then fail hard if the location doesn't exist? I worry about missing a backport or a typo in the internal scripts or something and having this silently fail. Thoughts?

---

I like that approach, will take a look at implementing